### PR TITLE
Pm 1551 bpay logos

### PR DESCRIPTION
**Pull request by d-bert**
#### List of Changes

Modifying BPay payment inquiry API to respond with ABI instead of correlationId.

#### Motivation and Context

Recovering and saving ABIs during BPay payment flow.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have created or updated an API interface.
  - [x] I have updated the OpenAPI spec
- [x] I have created or updated a database table
  - [x] I have updated both the Oracle and Postgres migrations
- [x] My PR is cohesive and self-contained (rule of thumb: less than 250 changed lines)

### CODE REVIEW

File: api-specs/api_spec_PGS.yaml [hunk 1]
```diff
@@ -107,20 +107,20 @@ paths:
...
+  /request-payments/bancomatpay/{transactionId}:
     get:
       tags:
         - BancomatPay
-      operationId: retrieve the correlationId of a BancomatPay payment request
+      operationId: retrieve the ABI of a BancomatPay payment request
       parameters:
         - in: path
-          name: requestId
+          name: transactionId
```
Comment (infantesimone) [inline api-specs/api_spec_PGS.yaml:117]: why rename in transactionId?

**Review by infantesimone — CHANGES_REQUESTED** [review]

File: api-specs/api_spec_PGS.yaml [hunk 2]
```diff
@@ -107,39 +139,6 @@ paths:
...
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/BPayInfoResponse'
-        "500":
-          description: Internal server error
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/BPayInfoResponse'
```
Comment (giovanniberti) [inline api-specs/api_spec_PGS.yaml:142]: Why was this removed?

Comment (d-bert) [reply api-specs/api_spec_PGS.yaml:142]: it's not been removed, it's been moved up under /request-payments/bancomatpay (because the path param has been converted to a query param)

File: api-specs/api_spec_PGS.yaml [hunk 3]
```diff
@@ -1180,12 +1179,12 @@ components:
     BPayInfoResponse:
       type: object
       properties:
-        correlationId:
```
Comment (giovanniberti) [inline api-specs/api_spec_PGS.yaml:1183]: Remove `correlationId` from the required properties

**Review by giovanniberti — APPROVED** [review]

**Review by infantesimone — APPROVED** [review]
